app/(app)/profile/page.tsx

```diff
@@ -0,0 +1,44 @@
+import { useTranslations } from "next-intl";
+import ProfileCard from "./_components/profileCard";
+import { Tabs, TabsContent, TabsList, TabsTrigger } from "@/components/ui/tabs";
+
+export const metadata = {
+  title: "My Profile",
+};
+
+const ProfilePage = () => {
+  const t = useTranslations("profile");
+
+  return (
+    <main className="px-[20px] mt-10 xl:mt-16 max-w-[935px] w-full">
+      <h1 className="unbound-font text-[20px] font-medium text-black xl:text-[34px] xl:font-bold xl:mb-6 mb-4">
+        {t("profile")}
+      </h1>
+      <ProfileCard />
+      <div className="mt-6">
+        <Tabs defaultValue="achievements">
+          <TabsList className="w-fit">
+            <TabsTrigger value="achievements">
+              <span className="unbound-font text-sm font-medium">{t("achievements")}</span>
+            </TabsTrigger>
+            <TabsTrigger value="certificates">
+              <span className="unbound-font text-sm font-medium">{t("certificates")}</span>
+            </TabsTrigger>
+          </TabsList>
+          <TabsContent value="achievements" className="xl:pt-4 pt-2">
+            <h6 className="unbount-font font-medium xl:font-normal text-[20px] xl:text-[24px] text-[#1A1A1A]">
+              {t("achievements")}
+            </h6>
+          </TabsContent>
+          <TabsContent value="certificates" className="xl:pt-4 pt-2">
+            <h6 className="unbount-font font-medium xl:font-normal text-[20px] xl:text-[24px] text-[#1A1A1A]">
+              {t("certificates")}
+            </h6>
```

Comment:
I think we could standardize every heading tag in globals (from h1 to h6) to default to:
- Use Unbound font
- Standard size and weights
- Use theme colors instead of explicit RGB

 We already have that in globals, but scoped by the `mdxeditor` class.
 
 Regarding the colors, we already have a lot of them, but not `#1A1A1A` specifically.
 We could add to globals and tailwind config like this:
 
 globals.css:
 ```
 :root
   --dark-gray: 0 0% 10%;
 ```
 
 tailwind.config.js:
 ```
 colors: {
   'dark-gray': 'hsl(var(--dark-gray))',
```

This way we automagically have a `text-dark-gray` tag and others like `bg-dark-gray`, `border-dark-gray`, and so on. This is better because this tags are more human-readable than RGB.